planner/core/logical_plans.go

```diff
@@ -634,7 +634,7 @@ func (ds *DataSource) Convert2Gathers() (gathers []LogicalPlan) {
 	tg := ds.buildTableGather()
 	gathers = append(gathers, tg)
 	for _, path := range ds.possibleAccessPaths {
-		if !path.IsTablePath() {
```

Comment:
It was a bug?

Reply:
Yes, but cascades planner isn't compatible with the cluster index, so it is not an important change.